src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -240,7 +240,8 @@ public function catalogAction(Request $request, $limit = 10, $offset = 0, $order
                 'enableSidebar' => true,
                 'help_link' => $this->generateSidebarLink('AdminProducts'),
                 'is_shop_context' => $this->get('prestashop.adapter.shop.context')->isShopContext(),
-                'permission_error' => $permissionError
+                'permission_error' => $permissionError,
+                'layoutTitle' => $this->trans('Products', 'Admin.Global'),
```

Comment:
minor issue, layoutTitle wasn't declared, and comes from AdminController (magically).